week-3/F-array-forEach/exercise.js

```diff
@@ -9,6 +9,23 @@
 
 var arr = new Array(100);
 
+for(var i = 1; i < arr.length; i++){
+  arr[i] = i;
+}
+
+arr.forEach(function(index) {
```

Comment:
The first parameter passed to `forEach` is the array item, not the index. The second parameter is the index.

So for this exercise you can get away with not filling the array up with values, and just use the index. (The first parameter is not used).

Reply:
Oh.. OK. I will try another way. It's just in MDN examples function in `forEach` has only one parameter. That's why I was confused.

Reply:
Also, when I tried without filling the array, the index in `forEach` was empty. I don't know, maybe I am wrong. I'll try the way you said.

Reply:
Just to clarify, do you mean I should use 2 parameters in `forEach` function?
like so `arr.forEach(function(arr, index)){};` ?

Reply:
Ah, you are correct. `forEach` does not iterate over empty array items.

It could be done this way http://jsbin.com/xupiwizaxo/edit?js,console but your original solution is fine.

Reply:
Now it's easy. Done!